README.md

```diff
@@ -1,15 +1,20 @@
-# Cancellation API
+# ECMAScript Cancellation
 
-This proposal defines an approach to user-controlled cancellation of asynchronous operations
+This proposal seeks to define an approach to user-controlled cancellation of asynchronous operations
 through the adoption of a set of native platform objects.
 
 ## Status
 
-**Stage:** 0  
-**Champion:** Ron Buckton (@rbuckton), Brian Terlson (@bterlson)
+**Stage:** 1  
+**Champion:** Ron Buckton (@rbuckton), Brian Terlson (@bterlson), Domenic Denicola (@domenic), Yehuda Katz (@wycats)
 
 _For more information see the [TC39 proposal process](https://tc39.github.io/process-document/)._
 
+> NOTE: TC39 has decided to accept that there is a need for a cancellation mechanism in the core library.
```

Comment:
I don't think that's an accurate characterization. We need to investigate the problem space, but not necessarily define a cancellation mechanism in the ECMAScript library.

Perhaps it would be more accurate to say that "TC39 has decide to investigate a cancelation mechanism in the core library".

Reply:
Thanks, I've updated the wording.